src/main/java/xyz/nucleoid/plasmid/widget/SidebarWidget.java

```diff
@@ -40,7 +46,10 @@ public static SidebarWidget open(Text title, PlayerSet players) {
         return widget;
     }
 
-    public void set(String[] display) {
+    /*
+       Style wont work.
+     */
+    public void set(TranslatableText[] display) {
```

Comment:
I’m not so sure about how we’ll want to set this out. I think it might make sense to somehow use our own text-representing object?